src/plugins/home/server/tutorials/elastic_serverless_forwarder_metrics/index.ts

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { i18n } from '@kbn/i18n';
+import { TutorialsCategory } from '../../services/tutorials';
+import {
+  onPremInstructions,
+  cloudInstructions,
+  onPremCloudInstructions,
+} from '../instructions/metricbeat_instructions';
+import {
+  TutorialContext,
+  TutorialSchema,
+} from '../../services/tutorials/lib/tutorials_registry_types';
+
+export function elasticsearchServerlessForwarderMetricsSpecProvider(context: TutorialContext): TutorialSchema {
+  const moduleName = 'elasticsearch-serverless-forwarder';
+  return {
+    id: moduleName + 'Metrics',
```

Comment:
metrics?

is it somehow a fixed value for both metrics and logs?

contrary currently the forwarder ships only logs

Reply:
This comment passed over. Sorry. I'm focusing on building the dev cycle: see a tile in the UI and then I'll write the texts. Those are the old from the copy-pasted tutorial